components/layout/stacked-link.tsx

```diff
@@ -29,7 +30,7 @@ export default function StackedLink(list: StackedListProps) {
               <a
                 className="helper-link-cover"
                 data-testid={`next-link-${link.title}`}
-                target="_blank"
+                target={isInternalLink(link.url) ? "_self" : "_blank"}
```

Comment:
This does nothing when the `Link` component still wraps the entire `<a>` tag, which will override whatever's made here.

We need to modify this element to be wrapped with `Link` or not based on whether or not the URL is internal or external. See [here](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/components/layout/navigation-menu.tsx#L68-L84) for example.